src/appleseed.qtcommon/utility/miscellaneous.cpp

```diff
@@ -310,6 +311,11 @@ QStringList get_open_filenames(
         settings_key + SettingsLastDirectory,
         QDir::toNativeSeparators(QFileInfo(filepaths.first()).path()));
 
+    for (int i = 0; i < filepaths.size(); ++i) 
+    {
+        filepaths[i].replace(0, filepaths[i].length(), QDir::toNativeSeparators(filepaths[i]));
```

Comment:
That looks fishy, can't you do this ?
```cpp
filepaths[i] = QDir::toNativeSeparators(filepaths[i]);
```

Reply:
Thanks for your review, all issues have addressed.